packages/mdc-layout-grid/_mixins.scss

```diff
@@ -48,6 +48,9 @@
 
   @supports (display: grid) {
     width: auto;
+    max-width: 100%;
+    max-height: 100%;
+    overflow: hidden;
```

Comment:
I ran into an issue while fixing this. If there's any other component inside the cell, then the child cell will still overflow as it has 100% max-height.

Check out this pen: http://codepen.io/vivekmarakana/pen/VpxqJa